lvmd/command/lvm_json.go

```diff
@@ -95,16 +95,12 @@ func (u *lv) UnmarshalJSON(data []byte) error {
 	u.uuid = temp.UUID
 	u.path = temp.Path
 
-	var convErr error
-	u.major, convErr = strconv.ParseUint(temp.Major, 10, 32)
-	if convErr != nil {
-		return convErr
-	}
-	u.minor, convErr = strconv.ParseUint(temp.Minor, 10, 32)
-	if convErr != nil {
-		return convErr
-	}
+	// If LV is not active, major/minor numbers will be -1, ignore conversion
+	// errors which results in 0 for values in this case.
+	u.major, _ = strconv.ParseUint(temp.Major, 10, 32)
+	u.minor, _ = strconv.ParseUint(temp.Minor, 10, 32)
```

Comment:
If only one of the numbers is -1, it is a different case from the issue. Would you please terminate the program in this case? And please add outputting a log if both of the numbers are -1.

Reply:
Do you really want to take down the service in this case?  Maybe just log it?  This change mimics previous behavior.

Reply:
I agree with @tasleson . Logging this would be better than having lvmd go into CrashLoopBackOff. 
We can also check with the LVM team if a scenario where only one number set to -1 can occur.

Reply:
@tasleson @nbalacha 
I'm aware that the current PR behaves the same as the version before using JSON. I have asked for the following reason.
I could not check the case is not exist where (major, minor)=(n, -1). In that case, I think we have a risk to access another LV because (major, minor)=(n, 0) is an acceptable device. If this PR is not appropriate for this change, I will make another PR. What do you think?

Reply:
Can you please make the change in another PR?

Reply:
@llamerada-jp @nbalacha From chatting with one of the lvm developers, the consensus is they will both be `-1` or both be valid major/minor numbers.

Reply:
@nbalacha @tasleson 
I see. Thank you for checking.